src/app/features/courses-page/components/course-search/course-search.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
feel free to move this component into `app/` folder. You can place feature modules directly there.

Reply:
@SteveRusin All components were moved to app/components folder with separate modules for each component.